azure-keyvault/azure/keyvault/key_vault_client.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         self.credentials = credentials
 
 
-class KeyVaultClient(object):
+class KeyVaultClient(SDKClient):
```

Comment:
@schaabs If you wonder what this is, I write that in my ChangeLog for this Autorest change:
> Client class can be used as a context manager to keep the underlying HTTP session open for performance

Which means that this:
```python
with KeyVault(params) as kv:
    kv.do_somthing()
```
will keep the same `requests.Session` object for the life of the `with`, and might improve a lot performance.
Not that in future async SDK, this syntax will be required since `aiohttp` force the `with` syntax (it's one of the reason I introduced it, I make the code more and more compatible with async so the switch is simple eventually).